### PR TITLE
docs: edi3 documentation

### PR DESCRIPTION
**Pull request by nissimsan**
Most of our old documentation. Will for sure require updating. Especially URLs pointing to edi3 need updating.

### CODE REVIEW

File: docs/tech-spec.md [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+```
+
+TODO: The formal process of assigning the business domain to the vocabulary terms is to be decided.
+
+### Versioning
+
+The vocabulary is updated every 6 months, following the maintainance cycle of the CEFACT BSP RDM.
+Each BIE is annotated with the date when it was created, current active\deprecated status, and the date of deprecation.
+
+```json
```
Comment (nissimsan) [inline docs/tech-spec.md:237]: @kshychko , @fak3 I don't think we're doing this any longer, are we?

Comment (Fak3) [reply docs/tech-spec.md:237]: Are you proposing not to do versioning, or annotation? Or you just state that current vocabulary inherited from edi3 work does not have version annotations?

Comment (nissimsan) [reply docs/tech-spec.md:237]: The latter - https://service.unece.org/trade/uncefact/vocabulary/uncefact/ doesn't seem to have it

Comment (Fak3) [reply docs/tech-spec.md:237]: Yes, this have not been implemented

**Comment by nissimsan** [comment]
@nissimsan , resolve conflict

**Pull request closed without merging**